src/common/components/CopyButton.tsx

```diff
@@ -16,12 +16,27 @@ export const CopyButton = ({
   );
 };
 
-export type ExtendedIconProps = Omit<IconProps, '_hover'> & {
+export type ExtendedIconProps = Omit<IconProps, '_hover' | '_groupHover' | 'color'> & {
   _hover?: ((copied: boolean) => IconProps['_hover']) | IconProps['_hover'];
+  _groupHover?: ((copied: boolean) => IconProps['_groupHover']) | IconProps['_groupHover'];
```

Comment:
it looks like this file reuse this pattern of omitting a prop then readding it as a function, you could create a generic type for this e.g. 
type CustomProps<T, K extends keyof T> = Omit<T, K> & {
   [P in K]?: ((copied: boolean) => T[P]) | T[P];
};

Reply:
I see. But I am not sure this makes the code any simpler or easy to understand